### ISSUE TITLE
Investigate Wheels of Lull Unofficial Patch

### ISSUE THREAD

**Issue by LostDragonist** [ModOrganizer2/modorganizer#940]
https://www.nexusmods.com/skyrimspecialedition/mods/19734?tab=posts

A sticky on this mod has some really tedious instructions for MO2 that I *can not* figure out what is supposed to be going wrong.  It seems like it should be "download, install, and set the priority" like every other mod out there.

Is there actually an issue here...?

Copied here for convenience: 

> -- MO2 Users Installation Guide --
> 
> As you can see from the comments here, the overwrite mechanism of MO2 combined with this mod causes some unfortunate CTDs at the start of the Wheels of Lull. Thankfully, with a few workaround steps this mod can be installed and managed with MO with no issues. The steps are:
> 
> 1) Download the Wheels of Lull SSE (https://www.nexusmods.com/skyrimspecialedition/mods/748), open the ZIP, and extract the contents to a temporary folder.
> 
> 2) If you don't have it installed, download and extract the Bethesda Archive Extractor Tool (https://www.nexusmods.com/skyrimspecialedition/mods/974/?) to a folder of your choosing.
> 
> 3) For each of the two BSA files included in your extracted Wheels of Lull folder:
> a) Launch the BAE executable
> b) Drag the file into the window as prompted
> c) Select the same folder the BSA was in to extract to
> d) Close the BAE utility and delete the BSA you have extracted.
> 
> 4) Download the Wheels of Lull Unofficial Patch, hosted on this page. Open up the ZIP, and copy the contents to your Wheels of Lull working folder, overwriting when prompted.
> 
> 5) If 1k Tintmasks or compatibility patches for The Forgotten City or Bruma are desired, download those and copy and overwrite those into the directory too.
> 
> 6) Create a new ZIP file called 'Wheels of Lull with Unofficial Patch' in the working folder and place all the files into the ZIP.
> 
> 7) Move the new ZIP file to your Mod Organizer/downloads directory and install it in the mod manager as you would any other mod.
> 
> 8) Once installed, click on the mod in the resource pane. If any other mods flash up green, this means that the Wheels of Lull ZIP will overwrite contents from the mod ingame. The mod we installed must be positioned high enough such that it is not overwriting any other mods - i.e. the mods that were showing green in the resource pane must show red when our Wheels of Lull mod is selected.
> 
> If no colours show for your other mods when selecting the Wheels of Lull mod, then it has not overwritten any other files, and you are safe.
> 
> 9) Sort your load order, do any Bashed / SSEEdit merged patches you like and you're all set!

**Comment by Silarn** [comment]
No, it should function like any other mod manager. I could see there being special instructions for MO1 with the special archive handling but this doesn't make sense to me other than someone doesn't know how to set up their priorities correctly.

The only reason to extract the BSA would be to bypass the BSA load order by creating loose files, which you can order by mod priority instead of plugin order.

**Comment by Silarn** [comment]
Of course this would overwrite any mods that use BSAs, so the final point there about not overwriting anything is actually not true.
